.github/workflows/tag.yml

```diff
@@ -8,7 +8,7 @@ on:
   push:
     # Sequence of patterns matched against refs/tags
     tags:
-      - "v[0-9]+.[0-9]+.[0-9]+*" # Push events to matching v*, i.e. v1.0, v20.15.10
+      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
```

Comment:
Can this method be used in cases such as `v1.0.0-rc1`?

Reply:
Please check the second explanation of PR.

Reply:
If then, we cannot use release note and assets in rc* tags? Oh, it is a problem.

Reply:
Then I will add it again.